locales/templates/app.pot

```diff
@@ -144,6 +144,23 @@ msgctxt "collection_separator_shots"
 msgid "%1$d shots"
 msgstr ""
 
+#. String displalyed when recognizing text on screenshot experiencing unknown
```

Comment:
typo: displayed

Assuming *recognize* is related to OCR, the noun is *recognition*.

Reply:
Did you still need to push? I'm not seeing any change for now